exercises/collatz-conjecture/example.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function collatzConjecture() {}
+
+collatzConjecture.prototype.steps = function(no){
+	var count = 0;
```

Comment:
Nested lines should be intended consistently from the surrounding one.  We're using the airbnb JS style guide, which calls for two spaces' indent: https://github.com/airbnb/javascript#whitespace--spaces